src/Plugin/AdvancedQueue/JobType/NetsEasyPaymentHandler.php

```diff
@@ -160,8 +156,8 @@ private function getPaymentAndSetRelevantValues(Job $job, WebformSubmissionInter
       $job->setPayload($payload);
 
     }
-    catch (\Exception | GuzzleException $e) {
-      $this->submissionLogger->error($this->t('The submission #@serial failed (@message)', [
+    catch (RuntimeException $e) {
```

Comment:
This will not catch `GuzzleException` (and other exceptions not extending your custom exception). You probably want to catch *all* exception and handle and log them. Or perhaps wrap then in a (new) custom exceptions, `ClientException`, say.